client/nginx.conf

```diff
@@ -185,6 +186,7 @@ server {
         add_header X-Frame-Options "SAMEORIGIN";
         add_header X-XSS-Protection "1; mode=block";
         add_header X-Content-Type-Options "nosniff";
+        add_header Content-Security-Policy "default-src 'self'; script-src 'self' https://accounts.google.com https://*.googleapis.com; connect-src 'self' http://localhost:* https://localhost:* http://127.0.0.1:* https://127.0.0.1:* https://yitam.org https://api.yitam.org https://api.anthropic.com https://accounts.google.com https://*.googleapis.com wss://yitam.org ws://yitam.org wss: ws:; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://accounts.google.com; font-src 'self' https://fonts.gstatic.com; img-src 'self' data: https://*.googleusercontent.com; frame-src 'self' https://accounts.google.com; object-src 'none'; base-uri 'self'; form-action 'self'; upgrade-insecure-requests;";
```

Comment:
Just remember that for ultimate security in your final production environment, you may want to remove the localhost entries once testing is complete.